### DISCUSSION TITLE
How to use this.$t on script section (i18n)

### DISCUSSION THREAD

**Discussion by wea141619** [quasarframework/quasar#11151] [General - Components / Directives / etc] [answered]
Hello guys I need you help, I implement the i18n to my app I dont have problems to refer to the tag in the HTML in this way and  works well.
<q-tab
          name="opt2"
          icon="construction"
          **:label="$t('formulation_tool')"**
 />

the issue happend when I try to refer the tag of the language in the section script like this.

<script>
export default {
  setup() {
    return {
      title: this.$q("message_title")
    };
  },
};
</script>

I got the error:  **Uncaught ReferenceError: $t is not defined**

Thanks for any help.

**Comment by MilosPaunovic** [comment]
Check the official docs:
[https://quasar.dev/options/app-internationalization](https://quasar.dev/options/app-internationalization)

**Reply by wea141619** [reply]
> Check the official docs: https://quasar.dev/options/app-internationalization

yes, I follow the documentation but I don't know what I do wrong.

**Comment by MilosPaunovic** [comment]
```js
this.$q("message_title")
```

Shouldn't this `$q` actually be `$t`?

**Reply by wea141619** [reply]
yes sorry was a mistake when I write the code, here you can see the original code 
![image](https://user-images.githubusercontent.com/44841569/139337687-3318e0fb-79a0-4eb8-b8c8-6fc1ed68d9b2.png)

**Comment by metalsadman** [comment, marked as answer]
https://vue-i18n.intlify.dev/guide/advanced/composition.html#message-translation

**Reply by wea141619** [reply]
thank you so much, with the documentation you send to me, works like a charm

**Comment by wea141619** [comment]
thank you guys for your help.
